tic_tac_toe/game.py

```diff
@@ -87,18 +137,48 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    if game['winner'] or _board_is_full(board):
+        raise InvalidMovement('Game is over.')
+    if player != game['next_turn']:
+        raise InvalidMovement('"{}" moves next.'.format(game['next_turn']))
+    if not _position_is_valid(position):
+        raise InvalidMovement('Position out of range.')
+    if not _position_is_empty_in_board(position, board):
+        raise InvalidMovement('Position already taken.')
+    board[position[0]][position[1]] = player
+    winner = _check_winning_combinations(board, player)
+    if winner:
+        game['winner'] = winner
```

Comment:
Instead of setting the variable `winner` and then two lines later assigning it to `game['winner']` you could just do `game['winner'] = _check_winning_combinations(board, player)` as `_check_winning_combinations` returns None if there is no winner. Also you don't need to set the `next_turn` to None as your code doesn't check to see if it is set to None, it just checks to see if the player trying to move is the same as `next_turn`